next-frontend/src/components/competitions/OrganizerCard.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+
+import { components } from "@/types/openapi";
+import {
+  Avatar,
+  Button,
+  Card,
+  Carousel,
+  HStack,
+  IconButton,
+  Stat,
+} from "@chakra-ui/react";
+import { Link as ChakraLink } from "@chakra-ui/react";
+import { MarkdownProse } from "@/components/Markdown";
+import WcaDocsIcon from "@/components/icons/WcaDocsIcon";
+import { LuChevronLeft, LuChevronRight } from "react-icons/lu";
+
+function OrganizerCarousel({
+  organizers,
+}: {
+  organizers:
+    | components["schemas"]["Person"][]
+    | components["schemas"]["Organizer"][];
```

Comment:
I think defining a (local, ie non-exported) type alias like `type OrganizerPerson = Person | Organizer` and then doing `OrganizerPerson[]` here is more intuitive.